scripts/build_wasi.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# We download the SDK
+if [[ ! -d wasi-sdk-5.0 ]]; then
```

Comment:
Can you put wasi-sdk-5.0 under `out` too?

Reply:
Good idea!